test/cpp/jit/backend_with_compiler.cpp

```diff
@@ -0,0 +1,111 @@
+#include <torch/csrc/jit/backends/backend.h>
+
+namespace torch {
+namespace jit {
+
+namespace {
+std::vector<std::string> parseBlob(const std::string& blob) {
+  std::vector<std::string> result;
+  std::stringstream s_stream(blob);
+  while (s_stream.good()) {
+    std::string substr;
+    getline(s_stream, substr, ',');
+    result.push_back(substr);
+  }
+  return result;
+}
+} // namespace
+// This test JIT backend is intended to do the minimal amount of work
+// necessary to test that the JIT backend registration endpoints and
+// code generation are working correctly. It is not intended to
+// produce numerically correct results.
```

Comment:
Maybe move this doc up there (see my other comment), so it's clear it covers both preprocess and the backend interface.